user-service/src/main/java/com/movie/recommender/user/model/dto/UserMapper.java

```diff
@@ -0,0 +1,20 @@
+package com.movie.recommender.user.model.dto;
+
+import com.movie.recommender.user.model.entity.User;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserMapper {
+
+    public User toEntity(UserDTO userDTO) {
```

Comment:
move the mapping methods into the `User` entity and DTOs classes themselves
(use `location service` as example)